### ISSUE TITLE
Remove Image Alt Text in Story Mode

### ISSUE THREAD

**Issue by HellGate94** [odota/ui#1171]
even if the image fails to load the text is there anyway and it would show duplicated text. this duplicated text also shows when you try to copy paste. example:

"Radiant won a teamfight by trading NecrophosNecrophos and Death ProphetDeath Prophet for RubickRubick, Shadow ShamanShadow Shaman, and UnderlordUnderlord, resulting in a net worth increase of 314314 gold."

where it should be:

"Radiant won a teamfight by trading Necrophos and Death Prophet for Rubick, Shadow Shaman, and Underlord, resulting in a net worth increase of 314 gold."

**Comment by howardchung** [comment]
I think lint rules require image elements to have alt text.

The images shouldn't be failing to load either, is this happening consistently?

**Comment by HellGate94** [comment]
html wants an alt text but it doesn't care if its empty so alt=""

also they don't fail to load but if they would then you would see the alt text + the manual text so its a duplicate like when you try to copy paste

**Comment by Steffo99** [comment]
This should be closed.
